.github/workflows/ci.yml

```diff
@@ -17,272 +17,323 @@
 #
 
 ---
-name: Airflow
+name: CI Build
 on:
+  schedule:
+    - cron: '10 2 * * *'
   push:
-    branches: ['*']
+    branches: ['master', 'v1-10-test', 'v1-10-stable']
   pull_request:
-    branches: [master]
+    branches: ['master', 'v1-10-test', 'v1-10-stable']
 env:
-  BUILD_ID: ${{github.sha }}
   MOUNT_LOCAL_SOURCES: "false"
-  MOUNT_HOST_AIRFLOW_VOLUME: "true"
   FORCE_ANSWER_TO_QUESTIONS: "yes"
   SKIP_CHECK_REMOTE_IMAGE: "true"
+  SKIP_CI_IMAGE_CHECK: "true"
   DB_RESET: "true"
   VERBOSE: "true"
-  CI: "true"
-  # Should be a target branch
-  TRAVIS_BRANCH: "master"
-  TRAVIS: "true"
+  UPGRADE_TO_LATEST_REQUIREMENTS: "false"
+  PYTHON_MAJOR_MINOR_VERSION: 3.6
+  # TODO: Enable cache registry once a master build succeeds uploading the images to Github registry cache
```

Comment:
This will work once we merge it to master. For now I leave it here because at the first master merge of that change we will build and push the CI images to Github Image registry and we will pull from there rather than from DockerHub. This should bring significant improvements on "Build CI Image" time of execution (around 5 minutes now)